ehitchhiking-ui/src/app/shared/components/no-data/no-data.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="no-data-wrapper">
+	<div class="no-data-block" [ngClass]="defineSize(size)">
+		<i class="material-icons">
+			{{ !this.icon ? 'block' : this.icon }}
```

Comment:
Pls create a constant for 'block'
this.icon || this.yourConstant

Reply:
fixed